pkg/workers/sharings/share_data.go

```diff
@@ -893,6 +916,41 @@ func getDirOrFileMetadataAtRecipient(id string, recInfo *RecipientInfo) (*vfs.Di
 	return dirDoc, fileDoc, nil
 }
 
+func headDirOrFileMetadataAtRecipient(id string, recInfo *RecipientInfo) error {
+	path := fmt.Sprintf("/files/download/%s", id)
+
+	_, err := request.Req(&request.Options{
+		Domain: recInfo.URL,
+		Scheme: recInfo.Scheme,
+		Method: http.MethodHead,
+		Path:   path,
+		Headers: request.Headers{
+			echo.HeaderContentType:   echo.MIMEApplicationJSON,
+			echo.HeaderAuthorization: "Bearer " + recInfo.Token,
+		},
+	})
+
+	if err != nil {
+		reqErr := err.(*request.Error)
```

Comment:
Are we sure this type assertion will not panic, ie: we do not have a request.Error ?
Maybe the `parseError` method could take an `error` type, and make a safe type assertion.